cmake/SuiteSparse-config-install.cmake.in

```diff
@@ -0,0 +1,36 @@
+# Compute locations from <prefix>/@{LIBRARY_DIR@/cmake/suitesparse-<v>/<self>.cmake
+get_filename_component(_SuiteSparse_SELF_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_SELF_DIR}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+
+# Load the LAPACK package with which we were built.
+find_package(LAPACK CONFIG)
+
+# Load targets from the install tree.
+if(NOT TARGET SuiteSparse::suitesparseconfig)
+  include(${_SuiteSparse_SELF_DIR}/SuiteSparse-targets.cmake)
```

Comment:
Can be just `include("${CMAKE_CURRENT_LIST_DIR}/SuiteSparse-targets.cmake")`, no need for `if(NOT TARGET ...)`.